packages/app/pages/organization/course/[cid]/edit.tsx

```diff
@@ -153,12 +152,10 @@ export default function Edit(): ReactElement {
         return
       }
 
-      // if semesterIdField is not a number or not in semesters
-      if (
-        isNaN(semesterIdField) ||
-        !semesters.find((semester) => semester.id === semesterIdField)
-      ) {
-        message.error('Semester is invalid')
+      if (semesterNameField.split(',').length !== 2) {
```

Comment:
might want to add a check to make sure that the second item is a number. I managed to cause an error by putting `nuh,uh` as my semester

Reply:
Fixed